src/plugins/fcrypt/fcrypt.c

```diff
@@ -260,6 +295,18 @@ static int fcryptGpgCallAndCleanup (Key * parentKey, KeySet * pluginConfig, char
 		{
 			shredTemporaryFile (parentKeyFd, parentKey);
 		}
+		if (manualCopy)
```

Comment:
Shouldn't we also shred in case of failed copies?

Reply:
It's handled in the "else"-branch one level above. But I see how I could simplify this code segment.